### PR TITLE
feat: Add video and slides to session

### LINKED ISSUE #5054: Public Session Page: Show social links, embed youtube or other video, add link to slides
The public session page should show the following additional items similar to generated website: http://2019.fossasia.org/event/schedule.html#5227

* social links
* embed video, e.g. youtube
* link to slides

Example for a session page: https://eventyay.com/e/8fa7fd14/session/6258

Screenshot from 2020-09-11 16-09-12

### PR DESCRIPTION
**Pull request by iamareebjamal**
image
image
image

Fixes #5054

### CODE REVIEW

File: app/utils/url.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { merge, mapValues, isArray } from 'lodash-es';
  * @param stringifyArray - Should an array be converted to csv
  * @return {string}
  */
-export const buildUrlViaUrlParse = (baseUrl, queryParams, stringifyArray = false) => {
+export const buildUrlViaUrlParse = (baseUrl: string, queryParams: { [key: string]: string }, stringifyArray = false) => {
```
Comment (niranjan94) [inline app/utils/url.ts:13]: Codacy found an issue: [Missing return type on function.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6186562)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6186562&bid=20189159)

File: app/utils/url.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const buildUrlViaUrlParse = (baseUrl, queryParams, stringifyArray = false
  * @param stringifyArray - Should an array be converted to csv
  * @return {string}
  */
-export const buildUrlViaQueryString = (baseUrl, queryParams, stringifyArray = false) => {
+export const buildUrlViaQueryString = (baseUrl: string, queryParams: { [key: string]: string }, stringifyArray = false) => {
```
Comment (niranjan94) [inline app/utils/url.ts:35]: Codacy found an issue: [Missing return type on function.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6186562)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6186562&bid=20189159)

**Review by mariobehling — APPROVED** [review]

**Pull request merged**